core/libp2p/crypto/common.hpp

```diff
@@ -60,8 +60,14 @@ namespace libp2p::crypto::common {
   /**
    * Supported types of all keys
    */
-  enum class KeyType : uint32_t { kUnspecified, kRSA1024, kRSA2048, kRSA4096, kED25519 };
-  // TODO:(yuraz) maybe add support for Secp256k1 like in js version
+  enum class KeyType {
+    kUnspecified,
+    kRSA1024,
+    kRSA2048,
+    kRSA4096,
+    kED25519
+  };
+  // TODO(yuraz): maybe add support for Secp256k1 like in js version
```

Comment:
Please, create a task for it and put its number here, if you are not going to do it in this PR

Reply:
done, will be implemented in PRE-103